nav2_rviz_plugins/src/utils.cpp

```diff
@@ -21,35 +21,19 @@ namespace nav2_rviz_plugins
 {
 
 void pluginLoader(
-  rclcpp::Node::SharedPtr node, bool & server_failed, const std::string & server_name,
+  rclcpp::Node::SharedPtr node, const std::string & server_name,
   const std::string & plugin_type, QComboBox * combo_box)
 {
   auto parameter_client = std::make_shared<rclcpp::SyncParametersClient>(node, server_name);
 
-  // Do not load the plugins if the combo box is already populated
-  if (combo_box->count() > 0) {
-    return;
-  }
-
   // Wait for the service to be available before calling it
-  bool server_unavailable = false;
   while (!parameter_client->wait_for_service(std::chrono::seconds(1))) {
     if (!rclcpp::ok()) {
       RCLCPP_ERROR(node->get_logger(), "Interrupted while waiting for the service. Exiting.");
       rclcpp::shutdown();
     }
-    RCLCPP_INFO(node->get_logger(), "%s service not available", server_name.c_str());
```

Comment:
Why remove? If this failed, shouldn't we log and notify that the server is unavailable to not try to call the client below it? 

I tseems like the server_unavailable should be maintained.